README.md

```diff
@@ -0,0 +1,65 @@
+# Bookwel-api :sparkles:
+This document outlines the functionalities of the Bookwel API, facilitating interactions with books, categories, users, reactions, and an AI-driven chat system.
+
+__NB__: This api is available at the following URL: https://book-api.preprod.hei.school
+
+## _Pre-requistes_ :sparkler:
+To launch this API locally, ensure you have the following prerequisites installed:
+
+    JDK - 17
+    Database
+
+Additionally, set the following environment variables:
+
+    AWS_ACCESS_KEY_ID
+    AWS_REGION
+    AWS_S3_BUCKET
+    AWS_SECRET_ACCESS_KEY
+    BOOK_RECOMMENDATION_API_URL
+    FIREBASE_DATA_CONFIG
+    OPENAI_API_KEY
+    SPRING_DATASOURCE_URL
+    SPRING_DATASOURCE_USERNAME
+
+## _Features_ :sparkler: 
+
+Note that some resources are protected. For more details, you can see an overview of the api specification [here](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/Bookwel-Team/bookwel-api/dev/doc/api.yml)
+
+### `/chats`
```

Comment:
As you already mention the OpenApi spec, I think it's not needed anymore to write all the endpoints here